src/reporters/console/console-reporter.js

```diff
@@ -27,9 +27,10 @@ type Row = Array<string>;
 export default class ConsoleReporter extends BaseReporter {
   constructor(opts: Object) {
     super(opts);
-    this._lastCategorySize = 0;
 
+    this._lastCategorySize = 0;
     this.format = (chalk: any);
+    this.isSilent = !!opts.isSilent;
```

Comment:
The intent is great, thanks for starting the PR.
The implementation confuses me because ConsoleReporter is used in other commands, not only in `run`.
In the 3 changed files I don't see this setting being used for only run command.

As a matter fact, I think silent could be as useful for other commands, too.
We just need to be aware of the effects.